mender/templates/api-gateway-configmap.yaml

```diff
@@ -417,10 +420,11 @@ data:
             regex: "^/api/management/v1/inventory/(.*)"
             replacement: "/api/0.1.0/$1"
 
+{{- if .Values.api_gateway.env.SSL }}
```

Comment:
```suggestion
{{- if $isTls }}
```

Reply:
Comments accepted and addressed in commit 53d7e06.